schema/show.js

```diff
@@ -71,7 +71,7 @@ const ShowType = new GraphQLObjectType({
       type: GraphQLString,
       description: 'The exhibition title',
       resolve: ({ name }) =>
-        isExisty(name) ? name.trim() : '',
+        isExisty(name) ? name.trim() : name,
```

Comment:
FYI, I think the JS idiom is to do `name && name.trim()`, but that’s not a blocker imo